javav2/example_code/timestream/src/main/java/com/timestream/write/DescribeTable.java

```diff
@@ -0,0 +1,79 @@
+//snippet-sourcedescription:[DescribeTable.java demonstrates how to return information about a database table.]
+//snippet-keyword:[AWS SDK for Java v2]
+//snippet-keyword:[Code Sample]
+//snippet-keyword:[Amazon Timestream]
+//snippet-sourcetype:[full-example]
+//snippet-sourcedate:[05-04-2022]
+//snippet-sourceauthor:[scmacdon- AWS]
+
+/*
+   Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+   SPDX-License-Identifier: Apache-2.0
+*/
+
+package com.timestream.write;
+
+//snippet-start:[timestream.java2.desc_table.import]
+import software.amazon.awssdk.regions.Region;
+import software.amazon.awssdk.services.timestreamwrite.TimestreamWriteClient;
+import software.amazon.awssdk.services.timestreamwrite.model.DescribeTableRequest;
+import software.amazon.awssdk.services.timestreamwrite.model.DescribeTableResponse;
+import software.amazon.awssdk.services.timestreamwrite.model.TimestreamWriteException;
+//snippet-end:[timestream.java2.desc_table.import]
+
+/**
+ * To run this Java V2 code example, ensure that you have set up your development environment, including your credentials.

```

Comment:
Before running this SDK for Java (v2) code example, set up your development environment, including your credentials.

Reply:
fixed

Reply:
2x checked